<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="newgame"/>
    <addaction name="BeginorStart"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>难度选择</string>
    </property>
    <addaction name="simple"/>
    <addaction name="normal"/>
    <addaction name="hard"/>
    <addaction name="veryhard"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="auto_1"/>
    <addaction name="auto_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newgame">
   <property name="text">
    <string>新游戏</string>
   </property>
  </action>
  <action name="BeginorStart">
   <property name="text">
    <string>开始/暂停</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="simple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>简单</string>
   </property>
  </action>
  <action name="hard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>困难</string>
   </property>
  </action>
  <action name="veryhard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>极难</string>
   </property>
  </action>
  <action name="normal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>普通</string>
   </property>
  </action>
  <action name="auto_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </action>
  <action name="auto_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
